eslint.config.mjs

```diff
@@ -0,0 +1,45 @@
+import typescriptEslintEslintPlugin from "@typescript-eslint/eslint-plugin";
```

Comment:
what's the difference between eslintrc and eslitn.config.mjs?

Reply:
It auto converted by the `npx @eslint/migrate-config .eslintrc` command. see https://eslint.org/docs/latest/use/configure/migration-guide.